### DISCUSSION TITLE
[New Feature] Add an option to control the movie playback volume.

### DISCUSSION THREAD

**Discussion by CCHyper** [Vinifera-Developers/Vinifera#1486] [Feature Requests]
### Description:
Add an option to control the movie playback volume.

### Possible Implementation:
A new INI key `MovieVolume` under the `[Audio]` section. This would default to `SoundVolume` and only override if found/set.

**Comment by Onidotmoe** [comment]
Would be nice to have this inside the game's sound settings too!
